Templates/Swift/template.yml

```diff
@@ -20,38 +20,33 @@ options:
     ID: String
     DateTime: Date
     File: Data
-  dependencies:
-    - name: Alamofire
-      pod: Alamofire
-      github: Alamofire/Alamofire
-      version: 4.9.0
-    - name: Poly
```

Comment:
I saw that we have added the `import Poly` manually in the model files. We should define the dependencies here and retrieve the values in the model files.

Reply:
@telip007 Ping

Reply:
I already removed the `import Poly` from the models since we include them as source code.

Reply:
Added changes.